.github/workflows/manual-aws-deploy.yml

```diff
@@ -0,0 +1,59 @@
+name: "Deploy AWS"
+
+on:
+  workflow_dispatch:
+    inputs:
+      prefix:
+        description: "Prefix for the deployment"
+        required: false
+        default: "armonik-cicd"
+      bootstrap:
+        description: "Deploy the S3 for the tfstate"
+        required: false
+        default: true
+      core-version:
+        description: "Version of core to deploy"
+        required: false
+        default: ""
+      gui-version:
+        description: "Version of gui to deploy"
+        required: false
+        default: ""
+      ext-csharp-version:
+        description: "Version of Extension Csharp to deploy"
+        required: false
+        default: ""
+
+jobs:
+  deploy:
+    name: "Deploy"
+    runs-on: ubuntu-latest
+    outputs:
+      terraform-output: ${{ steps.deploy.outputs.terraform-output }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install Dependencies
+        uses: aneoconsulting/ArmoniK.Action.Deploy/dependencies@main
+        with:
+          aws: true
+          terraform: true
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: eu-west-3
+      - if: ${{ inputs.bootstrap && inputs.bootstrap != 'false' }}
```

Comment:
Why two times inputs.bootstrap ?

Reply:
It was needed in the custom action because inputs are not typed there. But here, inputs are typed so we can actually remove it. Fixed.